src/genai/request/__init__.py

```diff
@@ -1,4 +1,13 @@
 """Modules containing functionalities related to text requests (their history)"""
 
 from genai.request.request_service import *
-from genai.request.schema import *
+
+
+def __getattr__(name: str):
+    """Allow additional imports for backward compatibility with old import system "from genai.service_name import"."""
+    if name in globals():
+        return globals()[name]
+
+    from genai._utils.deprecated_schema_import import _deprecated_schema_import
+
+    return _deprecated_schema_import(name, __name__)
```

Comment:
This function is duplicated multiple times.

Reply:
I know, I did it intentionally, it's possible to do:
```
def __getattr__(name, str):
    _deprecated_schema_import(name, __name__)
```
and move the `name in globals()` logic there, but I think it's more readable what actually happens when it's duplicated this way.

Do you think I should simplify it? (I expect this will be removed after the deprecation "expires")

Reply:
I see, so keep it as. But we should just mention in the code that we are going to remove it in the next major version so we do not forget that.